### DISCUSSION TITLE
How to aggregate same classes in different screens ?

### DISCUSSION THREAD

**Discussion by alamenai** [tailwindlabs/tailwindcss#1630] [Help]
I have this in my class :

```
cell-date p {
  @screen xs {
    @apply pl-6;
  }
  @screen sm {
    @apply ml-6;
  }
  @screen xl {
    @apply ml-8;
  }
}
```
Is there way to aggregate xs and sm and use one class , for example : 
```
@screen xs, sm{
@apply ml-6
}
```

**Comment by thecrypticace** [comment]
If your screens are min-width based then `@screen xs {…}` will already do what you want.
However, this could still be useful in situations where screens have been customized or where the rules have to apply to discontiguous screen sizes (e.g. xs and md but not sm because it already has separate styles applied).

ping @adamwathan If this is something you'd want I've already got the code and tests written for this and can open a PR (the existing setup made it like a 2 minute change ¯\\\_(ツ)\_/¯).

**Comment by alamenai** [comment]
@thecrypticace , yes I've customized my screens using intervals : 
```

screens: {
            'xs': {
                'min': '320px',
                'max': '659px'
            },
            'sm': {
                'min': '640px',
                'max': '767px'
            },
            'md': {
                'min': '768px',
                'max': '1023px'
            },
            'lg': {
                'min': '1024px',
                'max': '1279px'
            },
            'xl': {
                'min': '1280px'
            },
        },
```
@adamwathan , do you have a suggestion  to ignore code duplication in different screen sizes?

**Comment by tmswr** [comment]
I do see a strong use case for the ability to aggregate multiple @screen sizes in one line of code.

**Comment by hunganhAtWhill** [comment]
bump
